scripts/run_brain_age_bids.py

```diff
@@ -86,7 +87,8 @@
         df = pd.DataFrame([])
         for subject in freesurfer_subjects_to_analyze:
             in_file = os.path.join(out_dir, subject, subject + "_predicted_age.tsv")
-            df = df.append(pd.read_csv(in_file, sep="\t"))
+            if os.path.isfile(in_file):
+                df = df.append(pd.read_csv(in_file, sep="\t"))
```

Comment:
Could you please make this raise an error if `--skip_missing` is not set and the file that should be there is missing:

```
            if os.path.isfile(in_file):
                df = df.append(pd.read_csv(in_file, sep="\t"))
            elif not args.skip_missing:
                raise FileNotFoundError(in_file)
```